evap/evaluation/models.py

```diff
@@ -453,8 +457,8 @@ def is_user_editor(self, user):
 
     def warnings(self):
         result = []
-        if self.state in ['new', 'prepared', 'editor_approved'] and not self.has_enough_questionnaires:
-            result.append(_("Not enough questionnaires assigned"))
+        if self.state in ['new', 'prepared', 'editor_approved'] and not self.general_contribution_has_questionnaires:
+            result.append(_("No questionnaires assigned to course"))
         if self.state in ['in_evaluation', 'evaluated', 'reviewed', 'published'] and not self.can_publish_grades:
```

Comment:
please add another warning where `all_contributions_have_questionnaires` is `False`, e.g. `Not all contributors have questionnaires`